src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -1341,6 +1342,63 @@ public AggregationResult build(Object data) {
     }
   };
 
+  public static final Builder<ExhgetWithVerResult<String>> EXHGETWITHVER_RESULT_STRING
+      = new Builder<ExhgetWithVerResult<String>>() {
+    @Override
+    @SuppressWarnings("unchecked")
+    public ExhgetWithVerResult<String> build(Object data) {
+      if (data == null) {
+        return null;
+      }
+      List<Object> result = (List<Object>)data;
+      if (result.size() == 0) {
+        return null;
+      } else {
+        String value = SafeEncoder.encode((byte[])result.get(0));
+        long version = (Long)result.get(1);
+        return new ExhgetWithVerResult<>(value, version);
+      }
+    }
+
+    @Override
+    public String toString() {
+      return "exhgetwithver";
+    }
+  };
+
+  public static final Builder<List<ExhgetWithVerResult<String>>> EXHMGETWITHVER_RESULT_STRING_LIST
+      = new Builder<List<ExhgetWithVerResult<String>>>() {
+    @Override
+    @SuppressWarnings("unchecked")
+    public List<ExhgetWithVerResult<String>> build(Object data) {
+      if (data == null) {
+        return null;
+      }
+      List<Object> result = (List<Object>)data;
+      if (result.size() == 0) {
+        return null;
+      } else {
+        List<ExhgetWithVerResult<String>> results = new ArrayList<ExhgetWithVerResult<String>>();
+        for (Object o : result) {
+          if (o == null) {
+            results.add(null);
+          } else {
+            List<Object> lo = (List<Object>)o;
+            String value = SafeEncoder.encode((byte[])lo.get(0));
```

Comment:
check list zise

Reply:
As long as the return value is not empty, it must be two elements, and the engine guarantees, so there is no need to do check.